### PR TITLE
6.0.0-m2 pre-release changes

### PR DESCRIPTION
**Pull request by sbesson**
- review `pom.xml` following the decisions in #33 
- update the `README.md`
- update `NEWS.md` to document changes in last release including current 6.0.0 milestones

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -193,10 +193,10 @@
...
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.7.0</version>
-        <!-- Require the Java 7 platform. -->
+        <!-- Require the Java 8 platform. -->
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
         </configuration>
```
Comment (rleigh-codelibre) [inline pom.xml]: Note the javase javadoc link can also be updated to 8.

Comment (sbesson) [reply pom.xml]: :+1: pushed in 607d9c4

**Comment by dgault** [comment]
Changes to pom, news and readme all look good. Everything builds and displays as expected. The news and readme info is all correct and reads well. Good to merge

**Comment by sbesson** [comment]
Now released as http://repo1.maven.org/maven2/org/openmicroscopy/ome-common/6.0.0-m2

**Pull request merged**